website/static/js/pages/sharing-page.js

```diff
@@ -24,6 +29,48 @@ $('body').on('nodeLoad', function(event, data) {
     }
 });
 
+$('.filters').filters({
+    items: ['.contrib', '.admin'],
+    callback: function (filtered, empty) {
+        if ($.inArray('.contrib', empty) > -1) {
+            $('#noContributors').show();
+        }
+        else {
+            $('#noContributors').hide();
+        }
+        if ($.inArray('.admin', empty) > -1) {
+            $('#noAdminContribs').show();
+        }
+        else {
+            $('#noAdminContribs').hide();
+        }
+        var isFiltered = $.inArray('.contrib', filtered) > -1;
+        ko.contextFor($('#contributors').get(0)).$parent.sortable(isFiltered);
```

Comment:
@sloria if I am understanding your question correctly, I think that it was just bad naming and two different .sortable()s.  This one was calling a viewModel method and the one in afterRender is the jQuery .sortable().  I have scrapped the one in the viewModel and am now only using .isSortable() with a subscribed function on it to listen for changes.